tests/run-macros/annot-concrete-class/Macro_1.scala

```diff
@@ -0,0 +1,16 @@
+import scala.annotation.MacroAnnotation
+import scala.quoted.*
+
+class addMethod extends MacroAnnotation:
```

Comment:
To make the intent of the macro clearer we should name it `implementAFoo`. Then we would have

```scala
@implementAFoo
class Foo extends AFoo
```